### DISCUSSION TITLE
Regular ISO for User Testing

### DISCUSSION THREAD

**Discussion by chrisdebian** [AerynOS/discussions#4] [General]
Hi, all (Ikey BDFL :-) )

It was great to see the recent [release](https://serpentos.com/download/), would it be possible to schedule more frequent releases "PreAlpha1..." so that users who can't code, can contribute by giving feedback? Probably not a "Nightly" at this point, but maybe "Weekly"? Getting the ISO running on real hardware, would help with bug reporting and feature requests. Will SOS have a bug reporting mechanism, i'm thinking of something like [Apport](https://wiki.ubuntu.com/Apport)?

Many thanks, team.

Chris.

**Comment by ikeycode** [comment]
Sure frequent releases should be a priority. Let's get on that.

**Reply by chrisdebian** [reply]
Thanks, Ikey.

**Reply by ikeycode** [reply]
Sorry this is still slow Chris, with the version stuff happening we should be able to make this a lot easier.

**Reply by chrisdebian** [reply]
That's great, Ikey. Hope to install it one day.

Cheers,

Chris

**Comment by orgcontrib** [comment]
After some minor tweeks, I managed to build and run the ISO on Fedora. It was a good excercise as I've been reminded of some [facts](https://0pointer.de/blog/projects/tmp.html) along the way. I think that is equaly important to add Serpent OS to [The Operating System information database](https://libosinfo.org/) as we already have it on [Repology](https://repology.org/repository/serpentos).

diff --git a/desktop/img.sh b/desktop/img.sh
index fc6a5a4..058c3f9 100755
--- a/desktop/img.sh
+++ b/desktop/img.sh
@@ -88,7 +88,7 @@ echo "Using compression type: $COMPRESSOR"
 
 WORK="$(dirname $(realpath $0))"
 echo ">>> workdir \${WORK}: ${WORK}"
-TMPFS="/tmp/serpent_iso"
+TMPFS="/var/tmp/serpent_iso"
 echo ">>> tmpfs dir \${TMPFS}: ${TMPFS}"

then I ran:

`sudo --preserve-env="HOME,PATH" ./img.sh -p gnome_pkglist`

and I've learned about the GNOME Boxes [issue](https://gitlab.gnome.org/GNOME/gnome-boxes/-/issues/450).

**Reply by orgcontrib** [reply]
A bit off-topic, Fedora related: [BZ2278123](https://bugzilla.redhat.com/show_bug.cgi?id=2278123).

**Reply by ermo** [reply]
> After some minor tweeks, I managed to build and run the ISO on Fedora. It was a good excercise as I've been reminded of some [facts](https://0pointer.de/blog/projects/tmp.html) along the way.

I can build ISOs on Fedora just fine using the default `/tmp` dir:

![iso-building-fedora](https://github.com/user-attachments/assets/f6eb5e81-0b2f-49ea-85c8-8c7b512de9ea)


The reason we use `/tmp` is speed. If you are building ISOs in a VM, you should allocate at least 16 GiB of RAM to it, so the iso building process can use 8 GiB on /tmp (I believe it is set to use 50% of available RAM by default).

I use `sudo -E` myself, but your use of long options is possibly nicer.

**Comment by ikeycode** [comment]
Chris I think I fixed this xD
